pkg/capacity/topology/nodes.go

```diff
@@ -204,13 +204,20 @@ func (nt *nodeTopology) List() []*Segment {
 func (nt *nodeTopology) Run(ctx context.Context) {
 	go nt.nodeInformer.Informer().Run(ctx.Done())
 	go nt.csiNodeInformer.Informer().Run(ctx.Done())
-	go nt.runWorker(ctx)
 
 	klog.Info("Started node topology informer")
 	<-ctx.Done()
 	klog.Info("Shutting node topology informer")
 }
 
+func (nt *nodeTopology) RunWorker(ctx context.Context) {
+	go nt.runWorker(ctx)
+
+	klog.Infof("Started node topology worker")
+	<-ctx.Done()
+	klog.Infof("Shutting node topology worker")
+}
```

Comment:
Same here.

Reply:
Now that we have simplified this function like this, one other enhancement becomes possible: we no longer need the separate `runWorker` function and can save one goroutine:
```
func (nt *nodeTopology) RunWorker(ctx context.Context) {
	klog.Info("Started node topology informer")
        for nt.processNextWorkItem(ctx) {
	}
	klog.Info("Stopped node topology worker")
}
```